packages/browser-repl/src/iframe-runtime/iframe-runtime.spec.ts

```diff
@@ -1,14 +1,15 @@
 import { IframeRuntime } from './iframe-runtime';
 import { expect } from '../../testing/chai';
-import { ServiceProvider } from '@mongosh/service-provider-core';
+import { ServiceProvider /* TODO: bson */ } from '@mongosh/service-provider-core';
```

Comment:
@mcasimir Do you have any idea why I can't import bson from service-provider-core? That package just imports it and exports it directly. This isn't a huge deal bc its just for a test...but I don't understand.

Reply:
Don't really have an explanation, what is the error?

Reply:
Says that the bson package is a function but that it doesn't have any of the attributes that you'd expect, like `ObjectId`

Reply:
It's easier to explain if you try it - I wonder if it might be a webpack issue? Or I'm doing something dumb. If you're ok with it as is, then i'll merge